### PR TITLE
handling no data UI for utilization  bars

### PR DESCRIPTION
**Pull request by PreetiW**
Card:
https://issues.redhat.com/browse/RHIROS-382

Tereza and Pau have approved the changes, discussion link: https://ansible.slack.com/archives/C01C1GVT3TP/p1636448820092600

image

### CODE REVIEW

File: src/store/entitiesReducer.js [hunk 1]
```diff
@@ -32,23 +31,28 @@ export const displayState = (data) => {
...
-            : (
-                <Link to={{ pathname: `/${inventoryId}` }}
-                    className={ `pf-link recommendations link-${inventoryId}` }>
-                    { data }
-                </Link>
-            )
+        isDeleted ? <span className='recommendations'>{ state === NO_DATA_STATE ? NO_DATA_VALUE : data }</span>
+            : state === NO_DATA_STATE ?
+                (
+                    <span>{ NO_DATA_VALUE }</span>
```
Comment (kgaikwad) [inline src/store/entitiesReducer.js:48]: @PreetiW - It seems that with `recommendations` CSS class [here](https://github.com/RedHatInsights/ros-frontend/blob/main/src/Routes/RosPage/ros-page.scss#L25), it adjusts the left padding.
One observation: with this change, when there will be multiple records, vertical alignment for column `Suggestions` will not be same for rows.

**Review by kgaikwad — CHANGES_REQUESTED** [review]
Thank you @PreetiW! 
Added one inline observation, please let me know your thoughts.

**Review by kgaikwad — APPROVED** [review]

**Comment by kgaikwad** [comment]
Thanks @PreetiW for quick updates.
Just a small suggestion - if JIRA ticket exists for a change request, it would be good to include ticket number in git commit message for a future reference.

**Comment by PreetiW** [comment]
> Thanks @PreetiW for quick updates. Just a small suggestion - if JIRA ticket exists for a change request, it would be good to include ticket number in git commit message for a future reference.

Thank you for the reminder @kgaikwad as I forgot to do it for this PR. I will surely do it in the future.

**Pull request merged**